StaticGUI.py

```diff
@@ -154,15 +154,22 @@ def runSimulation(self, smoothPath):
 class SmoothedPathGUI(MapPathGUI):
     def __init__(self, master, inputMap, path):
         super().__init__(master, inputMap, path)
+        self.prev_line_id = []
```

Comment:
id rename to prev_line_ids

whenever you have a collection of something it is good practice to make the varaible name plural